src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1691,12 +1693,14 @@ static inline fsal_status_t kvsfs_setattrs(struct fsal_obj_handle *obj_hdl,
 		/* Truncate is a special IO-related operation */
 		result = kvsfs_ftruncate(obj_hdl, state, bypass, &stats, flags);
 	} else {
+		rc = cfs_fh_from_ino(obj->cfs_fs,
```

Comment:
Temporary created FH needs to be destroyed, please do that.

Reply:
Done.